pkg/controller/statusmanager/status_manager.go

```diff
@@ -113,6 +114,16 @@ func New(client cnoclient.Client, name string) *StatusManager {
 		ssInformers:  map[string]cache.SharedIndexInformer{},
 		ssListers:    map[string]StatefulSetLister{},
 	}
+	cluster := ""
+	if status.hyperShiftConfig.Enabled {
+		cluster = status.hyperShiftConfig.Name
```

Comment:
Should this be `bootstrapResult.Infra.InfraName`?
I noticed infraName is assigned to `names.GenerateStatusLabel` label.

Reply:
yes, absolutely. 
It is just a leftover from my previous approach

Reply:
Done, the PR is ready for a review @zshi-redhat